inspirehep/submissions/views.py

```diff
@@ -312,22 +330,14 @@ def create_ticket(self, record, rt_template):
             "job_url_edit": JOB_EDIT,
             "hep_url": INSPIREHEP_URL,
         }
-        try:
-            ticket = create_ticket_with_template(
-                rt_queue,
-                requestor,
-                rt_template,
-                rt_template_context,
-                f"Job {control_number} has been submitted to the Jobs database",
-                control_number,
-            )
-            if ticket == -1:
-                logger.error(
-                    "Cannot create RT ticket "
-                    "`create_ticket_with_template` returned `-1`!"
-                )
-        except AttributeError as e:
-            logger.error(f"RT is not initialized properly: {e}")
+        async_create_ticket_with_template(
+            rt_queue,
+            requestor,
+            rt_template,
+            rt_template_context,
+            f"Job {control_number} has been submitted to the Jobs database",
+            control_number,
+        )
```

Comment:
what about the test for this call?

Reply:
there are tests that asserts `create_ticket_with_template` called already, which are passing. However we need to improve them, by asserting the params too which isn't related to making the call `async`.